tooling/noir_js_backend_barretenberg/src/index.ts

```diff
@@ -1,25 +1,25 @@
 /* eslint-disable  @typescript-eslint/no-explicit-any */
 import { acirToUint8Array } from './serialize.js';
-import { Backend, CompiledCircuit, ProofData } from '@noir-lang/types';
+import { Backend, BackendOptions, CompiledCircuit, ProofData } from '@noir-lang/types';
 
 // This is the number of bytes in a UltraPlonk proof
 // minus the public inputs.
 const numBytesInProofWithoutPublicInputs: number = 2144;
 
 export class BarretenbergBackend implements Backend {
-  // These type assertions are used so that we don't
-  // have to initialize `api` and `acirComposer` in the constructor.
-  // These are initialized asynchronously in the `init` function,
-  // constructors cannot be asynchronous which is why we do this.
```

Comment:
Why is this removed?

Reply:
Those seemd to be in wrong line

Reply:
yep somehow it says I've deleted wrong lines but if you see the code, they're actually there